src/components/browserSupport/hooks/useBrowserSupport.js

```diff
@@ -80,7 +89,19 @@ export default (options = {}) => {
 
     log.debug({ isSupported })
 
-    isSupported ? handleSupported() : handleUnsupported()
+    if (isSupported && checkOutdated && isIOSWeb && isSafari) {
+      // TODO: check iOS version
+      const iosVersion = osVersion
+      const majoriOSVersion = iosVersion
+        .replace('iOS ', '')
+        .split('.')
+        .shift()
+      if (Number(majoriOSVersion) < 12) {
```

Comment:
pls move version to the config and add env variable.
so it could be re used here and in the outdated popup (we need to show which exact version is minimal to be supported)